examples/rdkafka_complex_consumer_example.cpp

```diff
@@ -81,7 +81,7 @@ static void print_time () {
         struct timeval tv;
         char buf[64];
         gettimeofday(&tv, NULL);
-        strftime(buf, sizeof(buf) - 1, "%Y-%m-%d %H:%M:%S", localtime(&tv.tv_sec));
+        strftime(buf, sizeof(buf) - 1, "%Y-%m-%d %H:%M:%S", localtime((const time_t*)&tv.tv_sec));
```

Comment:
Is this cast really needed?
If so, format as: `(const time_t *)&tv...`  (add space)

Reply:
I just remembered to double check this. The error mingw gives is `error: cannot convert 'long int*' to 'const time_t* {aka const long long int*}' for argument '1' to 'tm* localtime(const time_t*)'`